tasks/main.yml

```diff
@@ -7,6 +7,8 @@
 - name: Find out reserved memory for the crash kernel
   command: cat /sys/kernel/kexec_crash_size
   register: kexec_crash_size
+  changed_when: False
+  check_mode: False
```

Comment:
would using "slurp" help here? https://github.com/linux-system-roles/kdump/pull/22/files#diff-d2414d4ec8ba189e1a244b0afc9aa81e

Reply:
Yes, it seem to work better. I use it now.